src/main/webapp/readfile.html

```diff
@@ -16,9 +16,19 @@ <h2>Read Configuration</h2>
     <div>
     <form id="config-search">  
       <p>Customer ID:</p>
-      <input type="text" id="customer-id"><br>
+      <input list="customer-ids">
+      <datalist id="customer-ids">
+        <option value="Customer 1">
+        <option value="Customer 2">
+        <option value="Customer 3">
+      </datalist>
       <p>Account ID:</p>
-      <input type="text" id="account-id"><br>
+      <input list="account-ids">
+      <datalist id="account-ids">
+        <option value="Account 1">
+        <option value="Account 2">
+        <option value="Account 3">
+      </datalist><br>
       <button type="submit">Search</button>
```

Comment:
this may also exceed the scope of this PR, but it might be nice to see these HTML forms be refactored at some point to improve code re-usability. I can see this being hard to maintain

Reply:
What do you mean by this?